src/frontends/paddle/tests/test_models/gen_scripts/generate_unstack.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# unstack paddle model generator
+#
+import paddle
+import numpy as np
+from save_model import saveModel
+import sys
+
+
+def unstack(name: str, x, axis):
+    paddle.enable_static()
+
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        x_node = paddle.static.data(name="x", shape=x.shape, dtype=x.dtype)
+        out = paddle.unstack(x_node, axis)
+        place = paddle.CPUPlace()
+        exe = paddle.static.Executor(place)
+        exe.run(paddle.static.default_startup_program())
+        outs = exe.run(feed={"x": x}, fetch_list=[out])
+        saveModel(name, exe, feedkeys=['x'], fetchlist=out, inputs=[x], outputs=outs, target_dir=sys.argv[1])
+
+    return outs
+
+
+def main():
+    dtype = np.float32
+    x = np.random.randn(2, 3, 4).astype(dtype)
+    unstack(name='unstack_1', x=x, axis=0)
+
+    dtype = np.int32
+    x = np.random.randn(2, 3, 4).astype(dtype)
+    unstack(name='unstack_2', x=x, axis=1)
+
+    dtype = np.int64
+    x = np.random.randn(3, 4).astype(dtype)
+    unstack(name='unstack_3', x=x, axis=-1)
+
```

Comment:
from paddle unstack spec, 'axis ' parameter could be optional, so we would better add one such test case(e.g. unstack(name='unstack_4', x=x, axis=None)).
and, please add one more test case to cover input 'x' tensor which contains dimension 1 besides unstack axis(e.g.     
x = np.random.randn(2, 1, 4).astype(dtype)
    unstack(name='unstack_5', x=x, axis=0))